src/pages/login-page/LoginPage.jsx

```diff
@@ -1,8 +1,67 @@
 import { TopNav } from "../../components";
-import { Link } from "react-router-dom";
```

Comment:
You have imported Link 2 times. Just remove the one.

Reply:
Red line means the code has been removed